examples/Facemesh-keypoints/sketch.js

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2023 ml5
+//
+// This software is released under the MIT License.
+// https://opensource.org/licenses/MIT
+
+let facemesh;
+let video;
+let faces = [];
+let options = { maxFaces: 1, refineLandmarks: false, flipHorizontal: false };
```

Comment:
@B2xx @UpKindLikeWater I am wondering whether having an options object in the examples is worth having or not. `flipHorizontal` seems useful, but perhaps something that ML5 should better do uniformly across all models (e.g. with the existing utils class)? @shiffman

Is there a drawback to having the default for `maxFaces` at a higher value?
When would one want to enable `refineLandmarks`? (can we do it always, or never?)

Reply:
I used to take the approach of mirroring the video while drawing - but in models that don't have a `flipHorizontal` property, that leaves one with either some more convoluted code that does the arithmetic on the result - or running the whole `draw` function mirrored, which then quickly runs into issues when displaying text 😅.

I am not sure how feasible this is to implement, but my preference would be to have a class that transparently flips the image of the underlying video element (so that you can use it even in continuous mode), e.g.:

```
let video;
let flippedVideo;

function setup() {
  video = createCapture(VIDEO);
  // ..
  flippedVideo = flipVideo(video);
  facemesh.detectStart(flippedVideo, gotFaces);
}

function draw() {
  image(flippedVideo, ...);
}
```

Or, a flipHorizontal() method for videos in p5.js? 🤔

Reply:
Hi @gohai and @shiffman, here's the [documentation](https://github.com/tensorflow/tfjs-models/tree/master/face-landmarks-detection/src/tfjs) of tensorflow.js face mesh models and it talks about all the options that we could use. 

I love the idea of giving a general flip video option for ml5.js, however, I'm not that sure about the feasibility of it.  Flipping the image of the underlying video element definetely works better for me. Maybe we could use cope() and scale() to realize that?

According to my tests, there is no drawback to have maxFaces to higher value.

As for the refineLandmarks option, the documentation says 
> refineLandmarks: Defaults to false. If set to true, refines the landmark coordinates around the eyes and lips, and output additional landmarks around the irises.

Since the users will set refineLandmarks to true when they needed the irises, which I think is not that common. Maybe we could add a comment for this option or let it always to false?

Reply:
Hi all! In terms of bodypose and handpose, I agree with @gohai that a function like `ml5.flipVideo` would work better. Trying to flip the video using the p5 functions has caused a lot of problems for me personally, and it would be nice to have a way to flip the webcam video with one line of code and not interfere with other things.

As far as implementing a function like `ml5.flipVideo(video)` I am imaging taking these steps:
1. Get the original video from the p5 video object (or directly if the user inputs an HTML video)
2. Create an HTML canvas and draw the video on it with the x-axis flipped.
3. Convert the HTML canvas into a media stream using `captureStream()`
4. Return the media steam ad a video or p5 video depending on the input type.

I am not 100% sure whether this would work, but I am willing to try to implement this.

Reply:
Agreed with all thoughts! @ziyuan-linn I think a good next step would be for you to open a new issue duplicating your comment here and your proposal. We can then also tag in and check with p5.js team to make sure we aren't doing redundant work (in case they have something like this planned.) Or maybe they would prefer we add something to p5 directly! (I would ask here, but I think it may be hard to follow this conversation buried in an inline code comment for a specific pull request). Also, I don't think we need to resolve the "flipVideo" question as part of facemesh since it applies across the board.

Thanks all!!!